recipe/migrations/cuda92_100_101.yaml

```diff
@@ -1,34 +0,0 @@
-migrator_ts: 1611736741
-__migrator:
-  kind:
-    version
-  migration_number:
-    1
-  build_number:
-    1
-  paused: true
```

Comment:
yes. let's leave this file here. 

cc @jaimergp IIRC

Reply:
Do we want to still encourage people to add old versions though? Thought we were concerned about CI getting overwhelmed. No strong feelings personally. Just interested in making the addition of newer CUDA versions easier 🙂

Reply:
Yes, we need this file for legacy CUDA, even if it's opt-in only and discouraged. If https://github.com/conda-forge/conda-smithy/pull/1494 goes through, CI overusage would be less of a problem.

Reply:
@isuruf do you have any thoughts here?

Reply:
We should make a smithy release with conda-forge/conda-smithy#1494 and then keep this config. I can do the smithy release today.

Reply:
I think the challenge with leaving this file here is we want to move to the new compilers as well. If we leave this, it won't work with the new compilers and I'm not exactly sure how we can make it use the old compilers. If someone has a suggestion on that latter point, that would be welcome 🙂 However if we can't come up with anything, I'm not sure it's a good idea to leave a file that doesn't work properly

Reply:
Some time has passed since the discussion above

As the `nvidia/cuda` images for these versions no longer exist, keeping this doesn't make sense. So going to go ahead with dropping it